common/lib/xmodule/xmodule/modulestore/search.py

```diff
@@ -6,7 +6,7 @@
 LOGGER = getLogger(__name__)
 
 
-def path_to_location(modulestore, usage_key):
+def path_to_location(modulestore, usage_key, full_path=False):
```

Comment:
Optional: This might be cleaner as a pair of methods, one of which finds the full path to the root, and the other of which parses that into `(course_id, chapter, section, position)`.